package/MDAnalysis/topology/PDBParser.py

```diff
@@ -203,6 +206,9 @@ def parse(self, **kwargs):
         except AttributeError:
             warnings.warn("Invalid atom serials were present, "
                           "bonds will not be parsed")
+        except RuntimeError:
+            warnings.warn("CONECT records was corrupt, "
+                          "bonds will not be parsed")
```

Comment:
Not sure I see where the RunTime error gets generated?

Reply:
inside this function [def _parse_conect(conect):](https://github.com/MDAnalysis/mdanalysis/blob/c01d82bb86b480c8b2ae87f36497e7fc0a726c84/package/MDAnalysis/topology/PDBParser.py#L414)